Source/SourceKittenFramework/Request.swift

```diff
@@ -328,7 +328,12 @@ public enum Request {
         }
         let response = sourcekitd_send_request_sync(sourcekitObject)
         defer { sourcekitd_response_dispose(response) }
-        return fromSourceKit(sourcekitd_response_get_value(response)) as! [String: SourceKitRepresentable]
+        if let value = fromSourceKit(sourcekitd_response_get_value(response)) {
+          return value as! [String: SourceKitRepresentable]
```

Comment:
I think this could be just as the following:

```swift
if let value = fromSourceKit(sourcekitd_response_get_value(response)) as? [String: SourceKitRepresentable] {
    return value
} else {
    return [String: SourceKitRepresentable]()
}

```

or `return fromSourceKit(sourcekitd_response_get_value(response)) as? [String: SourceKitRepresentable] ?? [:]`?